### ISSUE TITLE
Enhance Navigation to navigate based on Filtered Index

### ISSUE THREAD

**Issue by bakerboy448** [Radarr/Radarr#4977]
Self explanatory.

When the index is filtered and user goes to navigate, we should navigate based on what is filtered, not all movies.

could possibly hide as an advanced UI toggle:?

[AB#782](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/782)

**Comment by rg9400** [comment]
If this is added, anyone that wants to navigate all movies can easily remove any applied custom filters, so navigation would work across all movies. On the other hand, there is no way for users to navigate a filtered index right now, so if they want to check something only available on the movie page, they have to navigate to a movie page, then back to the index, and then to the next movie.

From that perspective, even if it seems more confusing, I think this behavior would make the feature more flexible than the current way it works.

**Comment by mophawka** [comment]
+1 for this feature as not everyone sorts a-z

**Comment by ulrick65** [comment]
From reading through this, it appears that it was changed and implemented, but it has not been and still shows open.

Any comments as to the status of this would be appreciated.  I often have filters on and need to navigate based on those filters.

Thanks very much.

**Comment by bakerboy448** [comment]
As this is open and no closed/merged PR references this - it is not implemented

**Comment by ulrick65** [comment]
> As this is open and no closed/merged PR references this - it is not implemented

Thanks for the reply.  I am not that familiar with the way GitHub things work exactly.  I saw that @austinwbest did something there back in March of 2022 that sounded like a fix is why I asked.

Is there any way to tell when it might get looked at?  I may have to run a second instance or something as it is becoming an issue for me on the way my library is grouped and updated.

Thanks.

**Comment by bakerboy448** [comment]
The primary development team is engaged in over five applications, including Lidarr, Prowlarr, Radarr, Readarr, Whisparr, and Sonarr. Currently, there is approximately one active developer contributing to these projects, with none of the contributors working on them full-time. Additionally, all members involved, whether in development or support roles, are volunteers and do not receive compensation for their contributions. Each team member balances their commitments alongside full-time jobs, family responsibilities, and other personal obligations.

As a result, the list of tasks to be completed across these projects, along with the various backends and modules, is extensive, while the available time for addressing these tasks is limited.

**Comment by ulrick65** [comment]
> The primary development team is engaged in over five applications, including Lidarr, Prowlarr, Radarr, Readarr, Whisparr, and Sonarr. Currently, there is approximately one active developer contributing to these projects, with none of the contributors working on them full-time. Additionally, all members involved, whether in development or support roles, are volunteers and do not receive compensation for their contributions. Each team member balances their commitments alongside full-time jobs, family responsibilities, and other personal obligations.
> 
> As a result, the list of tasks to be completed across these projects, along with the various backends and modules, is extensive, while the available time for addressing these tasks is limited.

Fair enough...just wondered if there was a list or something that showed things being worked on, is all.  Appreciate all the work required for such an excellent program.  Nothing better out there in my opinion, which I why I have donated and will continue to do so again in the future.

Thanks.

**Comment by austinwbest** [comment]
> > As this is open and no closed/merged PR references this - it is not implemented
> 
> Thanks for the reply.  I am not that familiar with the way GitHub things work exactly.  I saw that @austinwbest did something there back in March of 2022 that sounded like a fix is why I asked.
> 
> Is there any way to tell when it might get looked at?  I may have to run a second instance or something as it is becoming an issue for me on the way my library is grouped and updated.
> 
> Thanks.

Yea, I did fix it years ago bud. Never got merged.

**Comment by ulrick65** [comment]
> > > As this is open and no closed/merged PR references this - it is not implemented
> > 
> > 
> > Thanks for the reply.  I am not that familiar with the way GitHub things work exactly.  I saw that [@austinwbest](https://github.com/austinwbest) did something there back in March of 2022 that sounded like a fix is why I asked.
> > Is there any way to tell when it might get looked at?  I may have to run a second instance or something as it is becoming an issue for me on the way my library is grouped and updated.
> > Thanks.
> 
> Yea, I did fix it years ago bud. Never got merged.

Dang...hate to hear that.  Thanks for the reply.

**Comment by ulrick65** [comment]
> > > As this is open and no closed/merged PR references this - it is not implemented
> > 
> > 
> > Thanks for the reply.  I am not that familiar with the way GitHub things work exactly.  I saw that [@austinwbest](https://github.com/austinwbest) did something there back in March of 2022 that sounded like a fix is why I asked.
> > Is there any way to tell when it might get looked at?  I may have to run a second instance or something as it is becoming an issue for me on the way my library is grouped and updated.
> > Thanks.
> 
> Yea, I did fix it years ago bud. Never got merged.

What does it take to get Merged?  This is really a pain not to be able to navigate between filtered media.  If the fix was already built, seems like it could be merged into at least the development branch or something.
